contrib/epee/src/net_helper.cpp

```diff
@@ -11,10 +11,39 @@ namespace net_utils
 		//////////////////////////////////////////////////////////////////////////
 		boost::asio::ip::tcp::resolver resolver(GET_IO_SERVICE(timeout));
 		boost::asio::ip::tcp::resolver::query query(boost::asio::ip::tcp::v4(), addr, port, boost::asio::ip::tcp::resolver::query::canonical_name);
-		boost::asio::ip::tcp::resolver::iterator iterator = resolver.resolve(query);
+
+		bool try_ipv6 = false;
+		boost::asio::ip::tcp::resolver::iterator iterator;
 		boost::asio::ip::tcp::resolver::iterator end;
-		if(iterator == end) // Documentation states that successful call is guaranteed to be non-empty
-			throw boost::system::system_error{boost::asio::error::fault, "Failed to resolve " + addr};
+    boost::system::error_code resolve_error;
```

Comment:
Can I ask to indent this as the surrounding code ? :)

Reply:
done